roles/ci_gen_kustomize_values/templates/edpm-values.j2

```diff
@@ -1,27 +1,28 @@
 data:
   ssh_keys:
-    authorized: |-
-{{ cifmw_ci_gen_kustomize_values_ssh_authorizedkeys | replace('\n', '\\n') | indent(6, first=True) }}
-    private: |-
-{{ cifmw_ci_gen_kustomize_values_ssh_private_key | replace('\n', '\\n') | indent(6, first=True) }}
-    public: |-
-{{ cifmw_ci_gen_kustomize_values_ssh_public_key | replace('\n', '\\n') | indent(6, first=True) }}
+    authorized: {{ cifmw_ci_gen_kustomize_values_ssh_authorizedkeys | b64encode }}
```

Comment:
ah, we can use b64 here? that's even better!

Reply:
Yeah, they are: https://github.com/openstack-k8s-operators/dataplane-operator/blob/836a066e4b7c6ec1bc4f689f044484445a0f67a1/api/v1beta1/common.go#L81-L89